### ISSUE TITLE
Add live exchange rate capability 

### ISSUE THREAD

**Issue by stevecondylios** [stevecondylios/priceR#11]
The [two current methods](https://stackoverflow.com/a/26694850) for retrieval of forex data in R are i) quantmod, which provides the current exchange rate (via Yahoo Finance), and ii) TFX for up to the moment rates, but the latter requires an account (albeit a free one). 

quantmod also provides historical rates with a 180 day limit (that may not be strictly enforced), and weekends missing from data, see [here](https://stackoverflow.com/a/28128770):

```R

library(quantmod)
# Use tryCatch() in case we try to get data too far in the past that 
# Oanda doesn't provide. Return NULL if there is an error, and Filter
# to only include data that has at least 1 row.
btc <- do.call(rbind, Filter(NROW, lapply(0:5, function(i) {
  tryCatch(getFX("BTC/USD", 
                 from = Sys.Date() -499 * (i + 1),
                 to = Sys.Date() - 499 * i,
                 env=NULL), error=function(e) NULL)
})))

symbols <- c(
  "UUP",
  "FXB",
  "FXE",
  "FXF",
  "FXY",
  "SLV",
  "GLD"
)
e <- new.env()
getSymbols(symbols, from=start(btc), env=e)
prices <- do.call(cbind, c(eapply(e, Cl)[symbols], list(btc)))
colnames(prices) <- gsub("\\.[A-z]*", "", colnames(prices))
head(na.locf(prices)[endpoints(prices, "months")])
```

Conclusion: Consider using IMF data via WB API (LCU per USD)

**Comment by stevecondylios** [comment]
[List of World Bank indicators](https://api.worldbank.org/v2/indicator?per_page=17500) - note: may increase/decrease over time - check param in url as it may need updating.

**Comment by stevecondylios** [comment]
Implemented in #79
